src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewControl.PrintPreviewControlAccessibleObject.cs

```diff
@@ -78,5 +88,135 @@ internal override UiaCore.IRawElementProviderFragmentRoot FragmentRoot
                     return base.FragmentNavigate(direction);
             }
         }
+
+        HRESULT IScrollProvider.Interface.Scroll(ScrollAmount horizontalAmount, ScrollAmount verticalAmount)
+        {
+            if (!this.TryGetOwnerAs(out PrintPreviewControl? owner))
+            {
+                return HRESULT.E_FAIL;
+            }
+
+            int scrollValue = 0;
+            if (owner._hScrollBar.Visible && horizontalAmount != ScrollAmount.ScrollAmount_NoAmount)
+            {
+                switch (horizontalAmount)
+                {
+                    case ScrollAmount.ScrollAmount_LargeIncrement:
+                        scrollValue = owner._hScrollBar.Value + owner._hScrollBar.LargeChange;
+                        owner._hScrollBar.Value = (scrollValue > owner._hScrollBar.Maximum ? owner._hScrollBar.Maximum : scrollValue);
+                        break;
+                    case ScrollAmount.ScrollAmount_SmallIncrement:
+                        scrollValue = owner._hScrollBar.Value + owner._hScrollBar.SmallChange;
+                        owner._hScrollBar.Value = (scrollValue > owner._hScrollBar.Maximum ? owner._hScrollBar.Maximum : scrollValue);
+                        break;
+                    case ScrollAmount.ScrollAmount_LargeDecrement:
+                        scrollValue = owner._hScrollBar.Value - owner._hScrollBar.LargeChange;
+                        owner._hScrollBar.Value = (scrollValue < owner._hScrollBar.Minimum ? owner._hScrollBar.Minimum : scrollValue);
+                        break;
+                    case ScrollAmount.ScrollAmount_SmallDecrement:
+                        scrollValue = owner._hScrollBar.Value - owner._hScrollBar.SmallChange;
+                        owner._hScrollBar.Value = (scrollValue < owner._hScrollBar.Minimum ? owner._hScrollBar.Minimum : scrollValue);
+                        break;
+                    case ScrollAmount.ScrollAmount_NoAmount:
+                        return HRESULT.E_FAIL;
+                }
+
+                return HRESULT.S_OK;
+            }
+
+            if (owner._vScrollBar.Visible && horizontalAmount != ScrollAmount.ScrollAmount_NoAmount)
+            {
+                switch (horizontalAmount)
```

Comment:
Should this be the following?
```suggestion
            if (owner._vScrollBar.Visible && verticalAmount!= ScrollAmount.ScrollAmount_NoAmount)
            {
                switch (verticalAmount)
```